idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -7,83 +7,16 @@ export default `
             <div class="idea-controls">
-              {{#model.enabled}}
-                <button data-hook="uninstall" class="button primary">Disable {{model.title}}</button>
-              {{/model.enabled}}
-              {{^model.enabled}}
-                <button data-hook="install" class="button primary">Enable {{model.title}}</button>
-              {{/model.enabled}}
-              <div class="user-count">7,654,321</div>
+              <button data-hook="uninstall" class="button primary">Disable <span data-hook="title"></span></button>
+              <button data-hook="install" class="button primary">Enable <span data-hook="title"></span></button>
             </div>
           </div>
         </div>
 
-        <div class="details-content">
-          <div class="details-overview">
-            <img src="{{model.thumbnail}}" width="260">
-            {{#model.measurements}}
-            <section class="measurement">
-              <h3>Measurements</h3>
-              <p class="disclaimer">All data is collected anonymously and used only to help us improve this test.</p>
-              {{{model.measurements}}}
-            </section>
-            {{/model.measurements}}
-            <section>
-              <h3>Brought to you by</h3>
-              <ul class="contributors">
-                {{#model.contributors}}
-                <li>
-                  <img class="avatar" src="{{avatar}}" width="56" height="56">
-                  <div class="contributor">
-                    <p class="name">{{display_name}}</span>
-                    <p class="title">{{title}}</span>
-                  </div>
-                </li>
-                {{/model.contributors}}
-                {{^model.contributors}}
-                  <!-- TODO: need a blank slate case for no contributors? -->
-                  <li>
-                    <img src="images/default-avatar@2x.png" width="56" height="56">
-                    <div class="contributor">
-                      <p class="name">To be announced</span>
-                    </div>
-                  </li>
-                {{/model.contributors}}
-              </ul>
-            </section>
-            <section>
-              <h3>Details</h3>
-              <table class="stats">
-                {{#model.version}}
-                <tr>
-                  <td>Version</td>
-                  <td>{{model.version}}{{#model.changelog_url}} <a href="{{model.changelog_url}}">changelog</a>{{/model.changelog_url}}</td>
-                </tr>
-                {{/model.version}}
-                <tr>
-                  <td>Last Update</td>
-                  <td>{{modified_date}}</td>
-                </tr>
-                <tr>
-                  <td>Contribute</td>
-                  <td><a href="{{model.contribute_url}}">{{model.contribute_url}}<a></td>
-                </tr>
-              </table>
-            </section>
-          </div>
+        <div data-hook="details"></div>
```

Comment:
Hmm, the more I look at this, the more I think all the stuff in the `details-view` could be merged into `experiment-page` rather than handled as a subview. But, I might be missing something...